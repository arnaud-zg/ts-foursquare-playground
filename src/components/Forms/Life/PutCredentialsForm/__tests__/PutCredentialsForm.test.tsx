import React from 'react'
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import { configureStore } from 'ts-foursquare'
import { PutCredentialsForm } from '..'

const store = configureStore({ middlewares: [] })

describe('Component/PutCredentialsForm', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <PutCredentialsForm />
        </Provider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
