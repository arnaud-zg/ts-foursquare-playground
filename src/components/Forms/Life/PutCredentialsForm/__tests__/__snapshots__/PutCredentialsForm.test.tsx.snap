// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/PutCredentialsForm renders correctly 1`] = `
<form
  className="max-w-xl m-2"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <p
    className="mt-4 text-gray-800 font-medium"
  >
    Credentials
  </p>
  <div
    className="mt-4 ml-4"
  >
    <label
      className="block text-sm text-gray-600"
      htmlFor="clientId"
    >
      clientId
    </label>
    <input
      className="mt-2 w-full px-5 py-2 text-gray-700 bg-gray-200 rounded"
      name="clientId"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Ex: 1234"
      type="text"
      value=""
    />
  </div>
  <div
    className="mt-4 ml-4"
  >
    <label
      className="block text-sm text-gray-600"
      htmlFor="clientSecret"
    >
      clientSecret
    </label>
    <input
      className="mt-2 w-full px-5 py-2 text-gray-700 bg-gray-200 rounded"
      name="clientSecret"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Ex: 1234"
      type="text"
      value=""
    />
  </div>
  <div
    className="mt-4"
  >
    <div
      className="flex justify-end"
    >
      <button
        className="bg-white text-gray-800 font-bold rounded border-b-2 border-green-500 hover:border-green-600 hover:bg-green-500 hover:text-white shadow-md py-2 px-6 inline-flex items-center"
        type="submit"
      >
        <span
          className="mr-2"
        >
          Send
        </span>
        <svg
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
            fill="currentcolor"
          />
        </svg>
      </button>
    </div>
  </div>
</form>
`;
